### DISCUSSION TITLE
Question: API limit

### DISCUSSION THREAD

**Discussion by 47ronin** [riesinger/plausible-exporter#15] [Q&A] [answered]
Would this be able to query Plausible's API (from Prometheus) and gather data from years ago?
(edit)
Also, any documentation on how exactly to bring this expoter's data into Prometheus? Docker is up and running with the site data pulling from Plausible's API into the container, but I don't see a method to connect this to Prometheus (also running in Docker). Thanks!

**Comment by riesinger** [comment, marked as answer]
Hi, thanks for your question! Currently, this is implemented in such a way that it requests the "latest" (daily) data from Plausible, not historic data. That's _usually_ how you'd run such an exporter. It's supposed to be always available and polled by Prometheus periodically. Querying historic data and moving it into Prometheus seems like something you'd do as a one-off, you wouldn't need to sync that again and again, hence I'm not sure this is useful for this project.

As for your second question: This works like any other Prometheus exporter. It exposes a `/metrics` endpoint that Prometheus should query. If you run both in Docker, make sure they're in the same network. I cannot help you any further since I don't know your setup.

Here are resources to get started with Prometheus:

* https://prometheus.io/docs/prometheus/latest/getting_started/
* https://grafana.com/docs/grafana/latest/getting-started/get-started-grafana-prometheus/

**Reply by 47ronin** [reply]
Thank you for the response. I appreciate it!
